cmake/OpenVDBHoudiniSetup.cmake

```diff
@@ -329,3 +329,50 @@ else()
   # Anticipated ABI version for H18
   set(OPENVDB_HOUDINI_ABI 6)
 endif()
+
+# ------------------------------------------------------------------------
+#  Configure GCC CXX11 ABI
+# ------------------------------------------------------------------------
+
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+  if((CMAKE_CXX_COMPILER_VERSION VERSION_EQUAL 5.1) OR
+     (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 5.1))
+    message(STATUS "GCC >= 5.1 detected. Configuring GCC CXX11 ABI for Houdini compatibility...")
+
+    execute_process(COMMAND echo "#include" " " "<string>"
+      COMMAND g++ "-x" "c++" "-E" "-dM" "-"
```

Comment:
It might be worth using `${CMAKE_CXX_COMPILER}` to avoid hard-coding `g++` here.

Reply:
done!